### PR TITLE
chore: load all the fast-components

### PR DESCRIPTION
**Pull request by radium-v**
# Description

Load fast-components for consumption on the website project.

## Motivation & context

This shoehorns all WCs available, we should probably reduce this from loading all to just the ones we need in the future.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-website/src/public/index.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (nicholasrice) [inline sites/fast-website/src/public/index.html:2]: can you also add `dir="ltr"`? I think we have a few components that look for that

File: sites/fast-website/src/app/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from "@microsoft/fast-components";
```
Comment (nicholasrice) [inline sites/fast-website/src/app/index.ts:1]: Does this get tree shaken if webpack has the production mode set?

Comment (radium-v) [reply sites/fast-website/src/app/index.ts:2]: I tested a prod build (see a450e0b) and confirm that webpack doesn't drop the exports and the components on the page load and work as expected.

Comment (nicholasrice) [reply sites/fast-website/src/app/index.ts:2]: Awesome, thanks!

**Pull request merged**